index.js

```diff
@@ -57,14 +57,14 @@ module.exports = function (input) {
   if (!validAmount(amount) || !parsableUnit(unit)) {
     throw 'Can\'t interpret ' + (input || 'a blank string');
   }
-  if (unit === '') return Number(amount);
+  if (unit === '') return Math.ceil(Number(amount));
```

Comment:
Sorry, I wasn't very clear.
if it is `1.0000000000000001`, it is almost certainly one of the infamous IEEE 754 floating point issues. There are logical extremes to which `2` might be more favorable, but I strongly believe that a vast majority of the time the rounded number is going to be a more valid answer. 

I am not going to merge in `ceil`, and I would prefer you get the credit for making the change as you discovered to bug. If you don't want to make the change, I understand, and encourage you to fork the project.

thank you, again, for reporting the issue